src/Lecture2.hs

```diff
@@ -198,7 +251,11 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge l1 [] = l1 
+merge [] l2 = l2 
+merge l1@(x1:xs1) l2@(x2:xs2) 
+  | x1 <= x2 = x1 : merge xs1 l2
+  | otherwise = x2 : merge l1 xs2
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️